### PR TITLE
Add get metadata functions

### LINKED ISSUE #87: Add "get metadata" API

### PR DESCRIPTION
**Pull request by elad-c**
## Pull Request Description:
Add "get metadata" API for Keras\Pytorch\ONNX models
Resolves #87 

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: mct_quantizers/keras/metadata.py [hunk 1]
```diff
@@ -58,9 +58,31 @@ def add_metadata(model: tf.keras.Model, metadata: Dict):
...
 
+    def get_metadata(model: tf.keras.Model) -> Dict:
+        """
+        Get the metadata dictionary from model.
+
+        Args:
+            model (Model): Keras model to extract metadata from.
+
+        Returns:
+            Model's the metadata dictionary.
```
Comment (ofirgo) [inline mct_quantizers/keras/metadata.py:69]: typo?

File: mct_quantizers/pytorch/metadata.py [hunk 2]
```diff
@@ -46,13 +46,35 @@ def add_metadata(model: torch.nn.Module, metadata: Dict):
...
 
+    def get_metadata(model: torch.nn.Module) -> Dict:
+        """
+        Get the metadata dictionary from model.
+
+        Args:
+            model (Module): Pytorch model to extract metadata from.
+
+        Returns:
+            Model's the metadata dictionary.
```
Comment (ofirgo) [inline mct_quantizers/pytorch/metadata.py:57]: same

File: mct_quantizers/pytorch/metadata.py [hunk 3]
```diff
@@ -82,9 +104,32 @@ def add_onnx_metadata(model: onnx.ModelProto, metadata: Dict):
...
+
+    def get_onnx_metadata(model: onnx.ModelProto) -> Dict:
+        """
+        Get the metadata dictionary from model.
+
+        Args:
+            model (ModelProto): onnx model to add the metadata to.
+
+        Returns:
+            Model's the metadata dictionary.
```
Comment (ofirgo) [inline mct_quantizers/pytorch/metadata.py:116]: same

**Review by ofirgo — APPROVED** [review]
